python/friesian/src/bigdl/friesian/utils/safepickle.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pickle
+import hmac
+import hashlib
+from bigdl.dllib.utils.log4Error import invalidInputError
+
+# Refer to this guide https://www.synopsys.com/blogs/software-security/python-pickling/
+# To safely use python pickle
+
+
+class SafePickle:
+    key = b'shared-key'
+    """
+    Example:
+        >>> from bigdl.chronos.utils import SafePickle
```

Comment:
maybe change to `from bigdl.friesian.utils import SafePickle`, other `safepickle.py` is similar.

Reply:
have update